src/utilities/codemirror/vml/vmlTreeUtils.test.ts

```diff
@@ -78,4 +78,93 @@ END_MODULE
     const cmdNode = vmlCommandInfoMapper.getContainingCommand(argNode);
     expect(vmlCommandInfoMapper.getArgumentAppendPosition(cmdNode)).toBe(inputPosition);
   });
+
+  test('identify parameters', () => {
```

Comment:
Would it be possible to reorganize these tests to be unit tests? It's a little difficult to determine which functions are being tested in each test. (This can be deferred to later)

Reply:
sounds good, the functional units will still likely include the whole sequence as input as substrings generally don't parse to subtrees

Reply:
That's fine. It's just a little hard to understand what each test is trying to do. Maybe for functional tests, can we add a `describe` around each one?